source/fundamentals/crud/read-operations/count.txt

```diff
@@ -51,12 +51,28 @@ Accurate Count
 --------------
 
 To count the number of documents that match your query filter, use the
-``CountDocuments()`` method.
+``CountDocuments()`` method. If you pass an empty query filter, this method
+returns the total number of documents in the collection.
 
 .. tip::
 
-   If you pass an empty query filter, this method returns the total
-   number of documents in the collection.
+   Using ``CountDocuments()`` to return the total number of documents in the
+   collection results in a collection scan. You can avoid a collection scan and
```

Comment:
Took this suggestion but slightly changed the wording since MongoDB does the collScan after the driver translates the query, rather than the driver doing it.